code-challenges.js

```diff
@@ -22,7 +24,28 @@ const fruit4 = "kiwi"
 
 // Pseudo code:
 
-const temperature1 = 42
+// Create a function that takes in a number parameter.
+
+// Evaluate the number with conditionals to determine if it is above, at, or below boiling point.
+
+// Use the provided boiling point temperature of 212 degrees as the threshold for comparison.
+
+// Return a string with the temperature and its relation to boiling point.
+
+const temperatureCheck = (temp) => {
+  if(temp<212) {
+    return "{temp} is above the boiling point"
```

Comment:
Nicely done! This is a good start and I think it just needs a small change. Look at your first condition being evaluated and the return. Then you can reference the syllabus for string interpolation syntax and you should be able to refactor this. 👍🏽